examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,125 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
```

Comment:
Can you please apply the changes from my comments here to ClusterExample.java as well?